packages/suite/src/views/wallet/send/components/Outputs/components/Amount/components/TokenSelect.tsx

```diff
@@ -49,35 +47,31 @@ export const buildTokenOptions = (
         },
     ];
 
-    if (tokens) {
-        const unknownTokens: Option['options'] = [];
-        const hasCoinDefinitions = getNetworkFeatures(symbol).includes('coin-definitions');
-
-        tokens.forEach(token => {
-            if (new BigNumber(token?.balance || '').eq('0')) {
-                return;
-            }
+    if (accountTokens) {
+        const tokens = getTokens(accountTokens, symbol, isDebug, coinDefinitions);
```

Comment:
does it make sense to have `isDebug` here? i dont think so

Reply:
Removed it and set to `false` by default